wasmplugin/config.go

```diff
@@ -28,9 +29,16 @@ func parsePluginConfiguration(data []byte) (pluginConfiguration, error) {
 	}
 
 	jsonData := gjson.ParseBytes(data)
-	jsonData.Get("rules").ForEach(func(_, value gjson.Result) bool {
-		config.rules = append(config.rules, value.String())
+	jsonData.Get("rules").ForEach(func(_, rule gjson.Result) bool {
+		config.rules = append(config.rules, rule.String())
 		return true
 	})
+
+	config.metricsLabels = make(map[string]string)
+	jsonData.Get("metrics_labels").ForEach(func(key, value gjson.Result) bool {
```

Comment:
Thanks! Just one small point left, I think I would call it `metric_labels`, sorry for not noticing earlier